src/dashboard.js

```diff
@@ -11,17 +9,17 @@
  * @copyright Julien Veyssier 2020
  */
 
-import Vue from 'vue'
-import './bootstrap.js'
```

Comment:
Is bootstrap gone and not needed anymore or is this removed by accident?

Reply:
boostrap is basically replaced by `Vue.mixin({ methods: { t, n } })`